tests/test_spaces.py

```diff
@@ -49,3 +49,20 @@ def test_identity_spaces(model_class, env):
     model.learn(total_timesteps=500)
 
     evaluate_policy(model, env, n_eval_episodes=5, warn=False)
+
+
+@pytest.mark.parametrize("model_class", [A2C, DDPG, DQN, PPO, SAC, TD3])
+@pytest.mark.parametrize("env", ["Pendulum-v0", "CartPole-v1"])
+def test_action_spaces(model_class, env):
+    if model_class in [SAC, DDPG, TD3]:
+        supported_action_space = env == "Pendulum-v0"
+    elif model_class == DQN:
+        supported_action_space = env == "CartPole-v1"
+    else:
```

Comment:
For clarity this could use a (dummy) check `elif model_class in [A2C, PPO]` to emphasize that these support both tested action spaces.